Exercise_C.js

```diff
@@ -14,5 +14,13 @@ See README.md if you need help running your code.
 */
 
 for (let i = 1; i <= 30; i++) {
-  // CODE HERE
+   if(i % 3 == 0 && i % 5 != 0) {
+     console.log ("Hong");
+   } else if(i % 5 == 0 && i % 3 != 0) {
+     console.log ("Kong");
+   }else if(i % 3 == 0 && i % 5 == 0) {
+     console.log ("Hong Kong");
+   }
+   console.log(i);
 }
+

```

Comment:
For this exercise, you have made use of && which can work at times but for this specific exercise, we have to make sure there is no remainder for 3 & 5 and then when it's 3 & 5. The thing that can be changed is you could use a common number like 15 to check if there is a remained and have the value become "Hong Kong" and for the numbers 3 and 5, they could be separate if statements.